File: BUEN_SABOR_BACKEND/src/main/java/com/lab4/buen_sabor_backend/model/Domicilio.java
package com.lab4.buen_sabor_backend.model;

import jakarta.persistence.*;
import lombok.*;
import com.fasterxml.jackson.annotation.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Domicilio extends Master{

    private String calle;
    private Integer numero;
    private Integer codigoPostal;
    private String piso;
    private String nroDepartamento;
    private String detalles;

    @ManyToOne
    @JoinColumn(name = "localidad_id")
    private Localidad localidad;
}

File: BUEN_SABOR_FRONTEND/src/models/Domicilio.ts
import Localidad from "./Localidad";

export default class Domicilio {
    id?: number = 0;
    calle: string = "";
    numero: number = 0;
    codigoPostal: string = "";
    localidad?: Localidad;
    piso?: string;
    nroDepartamento!: string;
    detalles?: string;
    eliminado!: boolean;
}